docs/html/dvr_autorec.html

```diff
@@ -51,6 +51,10 @@
 <dd>
 The name you've given to the rule, e.g. 'Stuff involving Jeremy Clarkson'.
 <p>
+<dt><b>Directory</b>
+<dd>
+The directory in which to save recordings to. Setting this will override any directory related settings from the DVR configuration. Useful for e.g. recording multiple different news broadcasts into one common directory.
```

Comment:
"The sub-directory into which to save recordings. Setting this will override any sub-directory related settings such as the creation of separate directories per day, channel, or title"